typo (#5871)
qmk/qmk_firmware: docs/feature_userspace.md

However, you can actually add support for keymap version, so that you can use it in both your userspace and your keymap! 

> For instance, lets look at the `layer_state_set_user` function.  Lets enable the [Tri Layer State](ref_functions.md#olkb-tri-layers) functionalitly to all of our boards, and then still have your `keymap.c` still able to use this functionality. 

Reviewer: ```suggestion
For instance, let's look at the `layer_state_set_user()` function.  You can enable the [Tri Layer State](ref_functions.md#olkb-tri-layers) functionality on all of your boards, while also retaining the Tri Layer functionality in your `keymap.c` files. 
```

I *think* this is what this paragraph is trying to say.

Author: Yep, that's better. I didn't even notice them. Thanks.
